internal/pkg/manifest/worker_svc.go

```diff
@@ -206,12 +208,36 @@ func NewWorkerService(props WorkerServiceProps) *WorkerService {
 		svc.WorkerServiceConfig.TaskConfig.CPU = aws.Int(MinWindowsTaskCPU)
 		svc.WorkerServiceConfig.TaskConfig.Memory = aws.Int(MinWindowsTaskMemory)
 	}
+	if len(props.Topics) > 0 {
+		setSubscriptionQueueDefaults(props.Topics, &svc.WorkerServiceConfig.Subscribe.Queue)
+	}
 	svc.WorkerServiceConfig.Subscribe.Topics = props.Topics
 	svc.WorkerServiceConfig.Platform = props.Platform
 	svc.parser = template.New()
 	return svc
 }
 
+func setSubscriptionQueueDefaults(topics []TopicSubscription, eventsQueue *SQSQueue) {
+	var isFIFOEnabled bool
+	for _, topic := range topics {
+		if strings.HasSuffix(aws.StringValue(topic.Name), ".fifo") {
```

Comment:
might be nice to add a function to `TopicSubscription` like `IsFIFO() bool` and use that instead? or maybe we add a bool to that field instead of relying on the Name so we don't have to rely on the topic "name" that we have to then trim off later? (L234)

Reply:
Good question Danny, So in case of some fifo topics and some not-fifo topics we have an opinion that we will create FIFO default events queue to which all the FIFO topics will be subscribing to. And all the other non-fifo topics will be having their own topic specific standard queue. 

Basically `isFIFOEnabled` variable tells us if we need to have a default FIFO Events queue or not. So in that we will still need to remove `.fifo` from the name in order for our manifest not to have `.fifo` named topics. 
does that make sense?

Reply:
> we have an opinion that we will create FIFO default events queue to which all the FIFO topics will be subscribing to

gotcha

> So in that we will still need to remove .fifo from the name in order for our manifest not to have .fifo named topics

i guess it just feels ...off that the name suffix and the type of topic are so mixed, but i get that's kinda just how it is and looks like already is assumed in other functions, so i think it's ok in these scenario as well